### ISSUE TITLE
[libsodium] DLLs do not have the App Container bit set

### ISSUE THREAD

**Issue by BestOwl** [microsoft/vcpkg#4330]
When I build libsodium:x86-uwp 1.0.16-1, i got this error
``` log
PS E:\Work\vcpkg> .\vcpkg.exe install libsodium:x86-uwp
The following packages will be built and installed:
    libsodium[core]:x86-uwp
Starting package 1/1: libsodium:x86-uwp
Building package libsodium[core]:x86-uwp...
-- Using cached E:/Work/vcpkg/downloads/jedisct1-libsodium-1.0.16.tar.gz
-- Using source at E:/Work/vcpkg/buildtrees/libsodium/src/1.0.16-60968e7afa
-- Applying patch E:/Work/vcpkg/ports/libsodium/disable-tests.patch
-- Applying patch failed. This is expected if this patch was previously applied.
-- Building E:/Work/vcpkg/buildtrees/libsodium/src/1.0.16-60968e7afa/libsodium.vcxproj for Release
-- Building E:/Work/vcpkg/buildtrees/libsodium/src/1.0.16-60968e7afa/libsodium.vcxproj for Debug
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/core.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_aead_aes256gcm.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_aead_chacha20poly1305.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_aead_xchacha20poly1305.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_auth.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_auth_hmacsha256.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_auth_hmacsha512.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_auth_hmacsha512256.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_box.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_box_curve25519xchacha20poly1305.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_box_curve25519xsalsa20poly1305.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_core_ed25519.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_core_hchacha20.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_core_hsalsa20.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_core_salsa20.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_core_salsa2012.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_core_salsa208.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_generichash.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_generichash_blake2b.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_hash.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_hash_sha256.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_hash_sha512.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_kdf.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_kdf_blake2b.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_kx.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_onetimeauth.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_onetimeauth_poly1305.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_pwhash.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_pwhash_argon2i.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_pwhash_argon2id.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_pwhash_scryptsalsa208sha256.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_scalarmult.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_scalarmult_curve25519.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_scalarmult_ed25519.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_secretbox.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_secretbox_xchacha20poly1305.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_secretbox_xsalsa20poly1305.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_secretstream_xchacha20poly1305.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_shorthash.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_shorthash_siphash24.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_sign.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_sign_ed25519.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_sign_edwards25519sha512batch.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_stream.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_stream_chacha20.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_stream_salsa20.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_stream_salsa2012.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_stream_salsa208.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_stream_xchacha20.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_stream_xsalsa20.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_verify_16.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_verify_32.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/crypto_verify_64.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/export.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/randombytes.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/randombytes_nativeclient.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/randombytes_salsa20_random.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/randombytes_sysrandom.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/runtime.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/utils.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/include/sodium/version.h
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/bin/libsodium.dll
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/debug/bin/libsodium.dll
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/lib/libsodium.lib
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/debug/lib/libsodium.lib
-- Installing: E:/Work/vcpkg/packages/libsodium_x86-uwp/share/libsodium/copyright
-- Performing post-build validation
The following DLLs do not have the App Container bit set:

    E:/Work/vcpkg/packages/libsodium_x86-uwp/debug/bin/libsodium.dll
    E:/Work/vcpkg/packages/libsodium_x86-uwp/bin/libsodium.dll

This bit is required for Windows Store apps.
Found 1 error(s). Please correct the portfile:
    E:\Work\vcpkg\ports\libsodium\portfile.cmake
-- Performing post-build validation done
Error: Building package libsodium:x86-uwp failed with: POST_BUILD_CHECKS_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: libsodium:x86-uwp
  Vcpkg version: 0.0.113-nohash

Additionally, attach any relevant sections from the log files above.
```

**Comment by BestOwl** [comment]
Look like is the same bug, #111

**Comment by wangli28** [comment]
Thanks for posting this issue. This issue has been fixed. I installed it successfully with VS2017 and VCPKG latest source. So I am closing this issue for now.

> PS E:\test\vcpkg> .\vcpkg.exe install libsodium:x86-uwp
The following packages will be built and installed:
    libsodium[core]:x86-uwp
Starting package 1/1: libsodium:x86-uwp
Building package libsodium[core]:x86-uwp...
-- Downloading https://github.com/jedisct1/libsodium/archive/1.0.17.tar.gz...
-- Extracting source E:/test/vcpkg/downloads/jedisct1-libsodium-1.0.17.tar.gz
-- Using source at E:/test/vcpkg/buildtrees/libsodium/src/1.0.17-2ff0ca6738
-- Configuring x86-uwp
-- Building x86-uwp-dbg
-- Building x86-uwp-rel
-- Performing post-build validation
-- Performing post-build validation done
Building package libsodium[core]:x86-uwp... done
Installing package libsodium[core]:x86-uwp...
Installing package libsodium[core]:x86-uwp... done
Elapsed time for package libsodium:x86-uwp: 48.89 s
/
Total elapsed time: 48.89 s
/
The package libsodium:x86-uwp provides CMake targets:
/
    find_package(unofficial-sodium CONFIG REQUIRED)
    target_link_libraries(main PRIVATE unofficial-sodium::sodium)
